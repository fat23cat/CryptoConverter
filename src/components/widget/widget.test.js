import React from 'react';
import ReactDOM from 'react-dom';
import CryptoWidget from './crypto-widget.js';

describe('components', () => {
  it('should render a CryptoWidget', () => {
   const wrapper = (<CryptoWidget />);
   expect(wrapper).toMatchSnapshot();
  })
})
