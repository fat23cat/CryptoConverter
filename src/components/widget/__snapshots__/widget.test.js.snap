// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components should render a CryptoWidget 1`] = `<CryptoWidget />`;
